src/Mollie/WC/Payment/Order.php

```diff
@@ -1086,4 +1090,20 @@ protected function updatePaymentDataWithOrderData($order, $orderId)
             $payment->update();
         }
     }
+
+    /**
+     * Method that shortens the field to a certain length
+     *
+     * @param $field
+     * @param int $maximalLength
+     *
+     * @return false|string
+     */
+    protected function maximalFieldLengths($field, $maximalLength)
+    {
+        if (is_int($maximalLength) && strlen($field) > $maximalLength) {
+            $field = substr($field, 0, $maximalLength);
```

Comment:
What will happen here if `$field` is an array for example?

SPOILER: it will throw a fatal error :_(